ServerRoomMonitoring.Generator/Services/GeneratorBackgroundService.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using ServerRoomMonitoring.Generator.Conditions;
+using ServerRoomMonitoring.Generator.Messaging;
+using ServerRoomMonitoring.Generator.Models;
+
+namespace ServerRoomMonitoring.Generator.Services
+{
+    public class GeneratorBackgroundService : BackgroundService
+    {
+        private ISensorQueue _queue;
+        private readonly  IStatus _status;
+        private IServerRoom _serverRoom;
+        
+        public GeneratorBackgroundService(ISensorQueue queue, IStatus status, IServerRoom serverRoom)
+        {
+            _queue = queue;
+            _status = status;
+            _serverRoom = serverRoom;
+        }
+        
+        protected override async Task ExecuteAsync(CancellationToken cancellationToken)
+        {
+            
+            while (!_status.Stopped)
+            {
+                foreach (var sensor in _serverRoom.Sensors)
```

Comment:
Przy takiej sekwencyjnej pętli, czasy opóźnień nie będą między wiadomościami dla jednego sensora. Wyśle pierwszy sensor, oczekiwanie pierwszego sensora, wyśle drugi sensor, oczekiwanie na drugi sensor itd. Zupełnie nie jest to równoległe.
Zamiast tego ten serwis powinien uruchomić każdy sensor w osobnym wątku np. `Task.Run` i każdy sensor powinien niezależnie od siebie generować wiadomości w takiej pętli.